src/config.ts

```diff
@@ -5,4 +5,4 @@ export const REACT_DIRNAME=process.env.REACT_APP_DIRNAME || '';
 export const REACT_ACCESSKEY=process.env.REACT_APP_ACCESS_KEY || '';
 export const REACT_SC=process.env.REACT_APP_SECRET_KEY || '';
 export const REACT_REGION=process.env.REACT_APP_REGION ||'';
-export const REACT_BACKEND_ROUTE=process.env.REACT_APP_BACKEND_URL || '';
\ No newline at end of file
+export const REACT_BACKEND_ROUTE=process.env.REACT_APP_BACKEND_ROUTE || '';
```

Comment:
we can use REACT_APP_BACKEND_URL instead of REACT_APP_BACKEND_ROUTE
where REACT_APP_BACKEND_URL=http://localhost:3000    in .env file